src/main.js

```diff
@@ -149,10 +147,36 @@ randomCoverButton.addEventListener('click',function() {
 
 makeYourOwnCoverButton.addEventListener('click', toggleToForm);
 
-viewSavedCoversButton.addEventListener('click', displaySavedCovers);
+
 
 homeButton.addEventListener('click', toggleToHome);
 
+saveCoverButton.addEventListener('click', function() {
+  saveUniqueCover(savedCovers);
+
+});
+
+viewSavedCoversButton.addEventListener('click', viewSavedCovers);
+
+function viewSavedCovers() {s
```

Comment:
@reneeswick  let's make sure to delete this stray s lingering after the function declaration and at the beginning of the function's code block we found together in this review. We can do that while we are working on the view saved covers functionality.